library/swift/Headers.swift

```diff
@@ -4,45 +4,58 @@ import Foundation
 /// To instantiate new instances, see `{Request|Response}HeadersBuilder`.
 @objcMembers
 public class Headers: NSObject {
-  let headers: [String: [String]]
+  let container: HeadersContainer
 
   /// Get the value for the provided header name.
   ///
-  /// - parameter name: Header name for which to get the current value.
+  /// - parameter name: The case-insensitive header name for which to
```

Comment:
Nit: I'd revert this to the previous description since there's nothing about this value that's case-insensitive, it's the lookup operation that's case-insensitive. I'd add a `- note:` to that effect instead.

Reply:
added note